src/app/organization/fund-mapping/fund-mapping.component.html

```
@@ -36,140 +36,125 @@
+
+      <mat-checkbox fxFlex="73%" labelPosition="before" formControlName="includeOutStandingAmountPercentage" class="margin-b">{{ 'labels.inputs.Loan Outstanding Percentage' | translate }}</mat-checkbox>
+
       <div *ngIf="fundMappingForm.value.includeOutStandingAmountPercentage" fxFlexFill fxLayout="row wrap" fxLayoutGap="2%" fxLayout.lt-md="column">
 
         <mat-form-field *ngIf="fundMappingForm.contains('outStandingAmountPercentageCondition')" fxFlex="31%">
-          <mat-label>Comparison Condition</mat-label>
+          <mat-label>{{ 'labels.inputs.Comparison Condition' | translate }}</mat-label>
           <mat-select required formControlName="outStandingAmountPercentageCondition">
-            <mat-option value="between">between</mat-option>
+            <mat-option value="between">{{ 'labels.inputs.between' | translate }}</mat-option>
             <mat-option value="<="><=</mat-option>
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)